ibm/mas_devops/roles/cp4d_service/tasks/gencfg/gencfg-wsl.yml

```diff
@@ -32,71 +32,71 @@
   set_fact:
     cpd_bearer_token: "{{ cpd_authorization.json.token }}"
 
+# Below logic to create WSL project is not needed anymore as this logic was moved to HPU operator
```

Comment:
Has that logic been backported to all the supported MAS versions?  If not then we need logic in here to handle the versions of MAS that still need this different to the ones that don't need it.